client/platform/web-girder/views/AnnotationTags.vue

```diff
@@ -0,0 +1,89 @@
+<script lang="ts">
+import {
+  defineComponent, ref,
+} from '@vue/composition-api';
+import {
+  useAnnotationTag,
+  useAnnotationTags,
+  useHandler,
+} from 'vue-media-annotator/provides';
+
+export default defineComponent({
+  name: 'AnnotationTags',
+  description: 'Annotation Tags',
+
+  setup() {
+    const currentTag = useAnnotationTag();
+    const tags = useAnnotationTags();
+    const newTag = ref('');
+    const validForm = ref(false);
+    const { save, tagChange } = useHandler();
+    const selectedTag = (tag: string) => {
+      tagChange(tag);
+    };
+
+    const addTag = async () => {
+      await save(newTag.value);
+      tagChange(newTag.value);
+    };
+
+    return {
+      currentTag,
+      selectedTag,
+      tags,
+      newTag,
+      addTag,
+      validForm,
+    };
+  },
+});
+</script>
+
+<template>
+  <div>
+    <p> Available Tags</p>
+    <v-list>
+      <v-list-item
+        v-for="tag in tags"
+        :key="tag"
+      >
+        <v-chip
+          outlined
+          :color="tag === currentTag || (!currentTag && tag === 'default') ? 'cyan' : 'white'"
```

Comment:
I think when someone navigates here it should show the `default` tag. It was a little surprising to have two tags appear when saving the first. Also then we likely wouldn't need to alert the user not to use default because its clearly already used.

Reply:
I'll add that in so that on initial navigation even with no tags it shows the 'default' tag in the list.